### PR TITLE
Request to add a permanent redirect for "BCI Ontology"

### PR DESCRIPTION
**Pull request by srodriguez142857**
As requested on:
http://lists.w3.org/Archives/Public/public-perma-id/2016Feb/0010.html

### CODE REVIEW

File: BCI-ontology/.htaccess [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ http://bci.pet.cs.nctu.edu.tw/ontology [R=302,L]
+RewriteRule ^v1$ ttp://bci.pet.cs.nctu.edu.tw/ontology/BCI.owl [R=302,L]
```
Comment (davidlehn) [inline BCI-ontology/.htaccess]: Change "ttp" to "http".

**Comment by davidlehn** [comment]
Oops, sorry that email request went unanswered.  Slipped through the cracks.  You might need to make the .htaccess file a bit more complex if you still wanted to add type detection.  But looks like you just changed the URL between base and owl endpoints so maybe it won't matter anymore.

**Comment by srodriguez142857** [comment]
Hello.

Thank you for your response. I have made the correction of the URL.
I would like to ask for your kind assistance in help me writing the .htaccess file to add the content-type detection, as requested on the email.

I really appreciate your help on this. Thank you.

Best regards,
Sergio.

**Comment by davidlehn** [comment]
I merged it as is for now.  I don't have time at the moment to help with content type negotiation.  You could look at what other people have done and experiment.

**Pull request merged**